pkg/reconciliation/construct_podtemplatespec.go

```diff
@@ -304,7 +304,14 @@ func addVolumes(dc *api.CassandraDatacenter, baseTemplate *corev1.PodTemplateSpe
 		},
 	}
 
-	volumeDefaults := []corev1.Volume{vServerConfig, vServerLogs}
+	vectorLib := corev1.Volume{
+		Name: "vector-lib",
+		VolumeSource: corev1.VolumeSource{
+			EmptyDir: &corev1.EmptyDirVolumeSource{},
+		},
+	}
+
+	volumeDefaults := []corev1.Volume{vServerConfig, vServerLogs, vectorLib}
```

Comment:
So is this always adding a Vector volume mount now? Even if cass-operator is deployed without Vector?

Reply:
Yes, but since it's not mounted anywhere it's not really created.